aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -86,33 +88,25 @@ std::unordered_map<CacheKey, cudnn_frontend::ManagedOpaqueDescriptor, ParamsHash
 
 }
 
-void raw_cudnn_convolution_forward_out(
-    const Tensor& output, const Tensor& input, const Tensor& weight,
-    IntArrayRef padding, IntArrayRef stride, IntArrayRef dilation, int64_t groups,
-    bool benchmark, bool deterministic, bool allow_tf32)
-{
-  TORCH_CHECK(!benchmark, "not supported yet");
-  if (output.numel() == 0) {
-    return;
-  }
-
-  cudnnHandle_t handle = getCudnnHandle();
-
-  CacheKey key;
-  setConvolutionParams(&key.params, input, weight, padding, stride, dilation, groups, deterministic, allow_tf32);
-  key.input_alignment = getAlignment(input);
-  key.output_alignment = getAlignment(output);
-  key.weight_alignment = getAlignment(weight);
+void get_cachekey(CacheKey& key, const cudnnBackendDescriptorType_t operation, const Tensor& convoutput, const Tensor& convinput1, const Tensor& convinput2, IntArrayRef padding, IntArrayRef stride, IntArrayRef dilation, int64_t groups, bool deterministic, bool allow_tf32) {
+   setConvolutionParams(&key.params, convinput1, convinput2, padding, stride, dilation, groups, deterministic, allow_tf32);
+   key.operation = operation;
+   key.input_alignment = getAlignment(convinput1);
+   key.output_alignment = getAlignment(convoutput);
+   key.weight_alignment = getAlignment(convinput2);
+}
```

Comment:
How about making variable names in `CacheKey` and `get_cachekey` consistent with `run_single_conv`?
`input`/`convinput1` --> `x`
`output`/`convoutput` --> `y`
`weight`/`convinput2` --> `w`